### DISCUSSION TITLE
Bosch BTH-RA doesn't work properly

### DISCUSSION THREAD

**Discussion by BikerGrisu** [Koenkk/zigbee2mqtt#18495] [Devices]
Hello,
I bought the Xiaomi SRTS-A01 thermostat months ago and am basically very satisfied with it. However, it has 2 disadvantages. The display cannot be read from above and the adjustment ring is very slippery.
So I looked around for a better thermostat and decided on the Bosch BTH-RA.
Unfortunately, I now find that the Bosch thermostat can be paired with zigbee2mqtt, but automation with Node-RED, as I am used to with the Xiaomi thermostat, does not work at all.
For example, a window contact should ensure that the state changes between "OFF" and "HEAT".
Maybe someone can tell me what else needs to be configured for the Bosch thermostat or recommend a better working thermostat.

Thanks, Frank Wiegmann

**Comment by Slomo17** [comment]
The window contact changes the Occupied heating setpoint to 8°C an does not turn it off completley, when turned off it set the setpoint back to the previous value
